x-pack/plugins/triggers_actions_ui/public/application/sections/rule_details/components/alert_summary/rule_alerts_summary.tsx

```diff
@@ -44,6 +45,7 @@ const G_ACCESSORS = ['g'];
 export const RuleAlertsSummary = ({ rule, filteredRuleTypes }: RuleAlertsSummaryProps) => {
   const [features, setFeatures] = useState<string>('');
   const isDarkMode = useUiSetting<boolean>('theme:darkMode');
+  const dateFormat = useUiSetting<string>('dateFormat');
```

Comment:
```suggestion
  const dateFormat = useUiSetting<string>('dateFormat').replace('@ HH:mm:ss.SSS', '');
```
We are only interested in dates without time. And it shows something weird for all bars.
### Before:
<img width="355" alt="Screenshot 2022-08-01 at 15 27 38" src="https://user-images.githubusercontent.com/6838659/182158490-6389e85d-010c-473a-b126-ec62a6cde0d8.png">


### After
<img width="314" alt="Screenshot 2022-08-01 at 15 28 35" src="https://user-images.githubusercontent.com/6838659/182158709-6fbabda6-9895-4401-a51d-bd0fb9686021.png">